### ISSUE TITLE
twisted.test.test_ssl has some unnecessary code

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5302]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#5302|
|Type|defect|
|Created|2011-10-09 20:39:11Z|
|Branch|https://github.com/twisted/twisted/tree/unnecessary-import-5302|

WriteDataTestCase is imported... and then deleted.

<details><summary>Searchable metadata</summary>

```
trac-id__5302 5302
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__branches_unnecessary_import_5302 branches/unnecessary-import-5302
branch_author__itamarst itamarst
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1318192751000000 1318192751000000
changetime__1318192959000000 1318192959000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_1"></a>|@itamarst commented|
|-|-|

(In [32806]) Branching to 'unnecessary-import-5302'

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst set status to `closed`|
|-|-|

(In [32808]) Merge unnecessary-import-5302.

Author: itamar
Review: exarkun
Fixes: [#5302](https://github.com/twisted/twisted/issues/5302)

Remove unnecessary import.
